lib/JSON/Validator.pm

```diff
@@ -315,6 +311,15 @@ sub _load_schema_from_url {
   return $self->_load_schema_from_text(\$tx->res->body);
 }
 
+sub _matcher {
+  my ($format, $module, $method) = @_;
+  my $e = eval "require $module;1" ? undef : $@;
+  return sub { warn "$module is not available: $e"; return undef }
+    if $e;
+  my $m = $module->can($method);
+  return sub { $m->($_[0]) ? undef : "Does not match $format format." };
+}
```

Comment:
Wouldn't this work with [Mojo::Loader::load_class](https://mojolicious.org/perldoc/Mojo/Loader#load_class)?

Reply:
I don’t think those modules have a new() method, so I wasn’t sure if it was awkward with load_class

Reply:
They both appear to have new() methods.